Scripts/linux/dtl

```diff
@@ -145,6 +145,14 @@ class DevTestLabsMain:
         self._printService.verbose('Parsed argument(s):')
         self._printService.verbose(self._parsedArgs)
 
+        if self._parsedArgs.clientID is None or self._parsedArgs.secret is None or self._parsedArgs.tenant is None:
+            self._printService.error('Missing required service principal credentials')
```

Comment:
It would be clearer here if you listed which args were required as none of them are actually called service principal